Birthday-2024-Project/Scripts/ScreenManagement/ScreenManager.gd

```diff
@@ -0,0 +1,51 @@
+class_name ScreenManager
+extends Node
+
+@export var popupRoot : CanvasLayer
+
+var lastScreen : String
+var _screenStack : Array[Node]
+
+func GoToScreen(screen : PackedScene, data : Dictionary):
```

Comment:
this needs to be looked into closer; using ScreenManager to load main_level messes up with pieces in level
![image](https://github.com/Saplings-Projects/Birthday-2024/assets/150629422/3a4a01f2-3a34-41fd-9be0-54510cc5c5ac)

Reply:
I encountered this issue and yet it completely slipped my mind. It's fixed now. Though, the solution re-introduced the issue with aspect ratios wider than 16:9 when on the level screen. Can probably tackle that later